packages/payment-processor/overrides/jest/index.d.ts

```
@@ -0,0 +1,1779 @@
+     * ${1} | ${2} | ${3}
+     * ${2} | ${1} | ${3}
+     * `('returns $expected when $a is added $b', ({a, b, expected}) => {
+     *    expect(a + b).toBe(expected);
+     * });
+     */
+    each: Each;
+  }
+
+  interface Describe {
+    // tslint:disable-next-line ban-types
+    (name: number | string | Function | FunctionLike, fn: EmptyFunction): void;
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)